CONTRIBUTING.md

```diff
@@ -24,3 +28,20 @@ Please note we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in
 ## How to set up your environment and run tests?
 
 1. Building and installation instructions are present in README.md file.
+
+## Coding Style
```

Comment:
Thanks for this 😊
This was long pending due. It would have been excellent if it was changed in another PR. But, it is absolutely fine for now😊